tests/task_loop/test_logging.py

```diff
@@ -173,7 +173,7 @@ def main():
         tl.log_debug("line 3")
 
         del tl.LogContext.dest_levels["varied"]
-        tl.LogContext.dest_levels[None] = "warning"
+        tl.LogContext.dest_levels[None] = "warning"  # type:ignore
```

Comment:
Why is `# type:ignore` needed here? I think `None` actually wouldn't be allowed here given the type signature for `dest_levels` so is there a reason to test it?

Reply:
Accidentally leftover from when I was using `None` as the default value